internal/trafficcontrol/tc.go

```diff
@@ -201,3 +212,33 @@ func IsSriovPF(ifName string) bool {
 	}
 	return true
 }
+
+func GetSriovPFInfo(ifName string) (*PFInfo, error) {
+	if !IsSriovPF(ifName) {
+		return nil, fmt.Errorf("interface %s is not a sriov pf type", ifName)
```

Comment:
Maybe we don't need to special-case this any more - Treat all PFs the same, and always populate a `PFInfo` - Just if a PF isn't set up with SRIOV, the VFInfo list will be empty.

Reply:
yes, removed it.